scripts/dpdk_setup_ports.py

```diff
@@ -48,7 +52,7 @@ class ConfigCreator(object):
     mandatory_interface_fields = ['Slot_str', 'Device_str', 'NUMA']
     _2hex_re = '[\da-fA-F]{2}'
     mac_re = re.compile('^({0}:){{5}}{0}$'.format(_2hex_re))
-    MAC_LCORE_NUM = 63 # current bitmask is 64 bit
+    MAX_LCORE_NUM = 63 # current bitmask is 64 bit
```

Comment:
I think this should be per platform make file configuration. The code is cross compiled and in any x86 machine that we compile the code we would like 64 maximum CORES

Reply:
Done.

Reply:
I would note that I removed the bitmask used to configure logical cores for the DPDK EAL, switching from the "-c" option to the "-l" option which uses a string.  The maximum lcore limit still applies, but will be simpler to raise in the future.